settings.gradle

```diff
@@ -41,3 +41,7 @@ include "code/korniakov-kirill-lab3/View"
 project(":code/korniakov-kirill-lab3/View").name = "korniakov-kirill-lab3-View"
 include "code/korniakov-kirill-lab3/Infrastructure"
 project(":code/korniakov-kirill-lab3/Infrastructure").name = "korniakov-kirill-lab3-Infrastructure"
+
+// Yuferev Kirill, lab1
+include "code/stack"
+project(":code/stack").name = "stack-Model"
```

Comment:
There is "No newline at end of file" message. Please fix it.

Reply:
it's not actual anymore)